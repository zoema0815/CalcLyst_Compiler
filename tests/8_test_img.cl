int main () {
    Imaginary a;
    Imaginary b;
    a = 3.2i;
    b = 1.3i;
    printImaginary(a);
    printImaginary(b);
    printImaginary(a + b);
    printImaginary(a - b);
    printReal(a * b);
    printReal(a / b);
    return 0;
}