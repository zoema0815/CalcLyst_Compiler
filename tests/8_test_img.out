3.2i
1.3i
4.500000i
1.900000i
-4.160000r
2.461539r
